### ISSUE TITLE
[Feature ]Can this be customized for any upcoming movie? 

### ISSUE THREAD

**Issue by ShreshthArora** [shubhank-saxena/endgame-spoiler-blocker#1]
What if it has options for upcomming movies which needs to be blocked and extension does the same work as in it has done with endgame

**Comment by shubhank-saxena** [comment]
Yes. It can be. It relies on a dictionary, which consists of various characters from that film.
